### PR TITLE
getting ready for AI for earth 

### PR DESCRIPTION
**Pull request by tam203**
Address issue #8.

### CODE REVIEW

File: README_data.md [hunk 1]
```diff
@@ -1,56 +1,63 @@
...
+- [Announcement by the Met Office](https://medium.com/informatics-lab/met-office-and-partners-offer-data-and-compute-platform-for-covid-19-researchers-83848ac55f5f) making this data available in response to [RAMP](https://epcced.github.io/ramp/) initiative, asking for assistance in tackling to the COVID-19 pandemic.
 
+- You can browse the data we have made available on Azure [here](index.html).
 
+- This data is made available under [Open Government License](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/).
 
-## Quick start.
+- More data is coming! Please subscribe to our [Google Groups mailing list](https://groups.google.com/forum/#!forum/met-office-covid-19-data-and-platform-updates/join) to receive updates when that data is available.
 
-### What data is available?
```
Comment (kaedonkers) [inline README_data.md:32]: Is it necessary to remove descriptions of what variables are available? I think it's important to specify exactly what the values mean

**Comment by tam203** [comment]
- [ ] re-instate section describing parameters
- [ ] add link to examples binder

assigned to @kaedonkers

**Comment by tam203** [comment]
No 100% done yet because want a link to examples but I think it's an improvement so merging.

**Comment by kaedonkers** [comment]
I didn't get a chance to look at it but we can always make another PR

**Comment by kaedonkers** [comment]
Thanks Theo!

**Pull request merged**